packages/components/src/Components/TagCount/tagCount.scss

```diff
@@ -1,5 +1,7 @@
 .ins-c-tag-count {
     color: var(--pf-global--icon--Color--light);
+    display: flex !important;
```

Comment:
We try not to ever use `!important`. You can always just increase the specificity of the parent tag if needed.